context/src/main/java/com/c8y/ms/templates/context/service/AlarmService.java

```diff
@@ -0,0 +1,32 @@
+package com.c8y.ms.templates.context.service;
+
+import com.cumulocity.microservice.subscription.service.MicroserviceSubscriptionsService;
+import com.cumulocity.rest.representation.alarm.AlarmRepresentation;
+import com.cumulocity.sdk.client.alarm.AlarmApi;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class AlarmService {
+
+    @Autowired
```

Comment:
I personaly like more autowiring into constructor instead of field autowiring. Constructure injection is easier to handle if you want to mock this services. However, i am fine with that.